tron/config/config_parse.py

```diff
@@ -249,6 +251,17 @@ class ValidateVolume(Validator):
 valid_volume = ValidateVolume()
 
 
+class ValidateSecretSource(Validator):
+    config_class = ConfigSecretSource
+    validators = {
+        "secret": valid_string,
```

Comment:
should we add some comments about what these keys will be holding? (i.e., `secret` isn't actually holding plaintext secrets)

Reply:
I will definitely add comments to help readers, but you think this confusing enough that the config key should be named more clearly, eg `secret_name`?

Reply:
it's probably fine as-is with the comment, but I wouldn't be against naming this key `secret_name`

Reply:
Agreed, i've done this and updated Paasta#3154 to match